src/hooks/useAssetLogoSource.ts

```diff
@@ -41,6 +43,14 @@ function getInitialUrl(address?: string | null, chainId?: number | null, isNativ
 
   const networkName = chainId ? chainIdToNetworkName(chainId) : 'ethereum'
   const checksummedAddress = isAddress(address)
+
+  // Celo logo logo is hosted elsewhere.
+  if (chainId && isCelo(chainId)) {
+    if (address === nativeOnChain(chainId).wrapped.address) {
```

Comment:
should there be an else case?

Reply:
<img width="867" alt="Screenshot 2023-06-05 at 3 59 08 PM" src="https://github.com/Uniswap/interface/assets/35351983/7d2d380d-43a3-4e38-b1db-1773b4d106a1">

I could make it an else if statement, but the default is just to return the normal location which is handled by the if statement below

Reply:
oh okay. then you can remove the nested if conditional?

Reply:
sounds good! just pushed changes